README.md

```diff
@@ -208,6 +208,12 @@ If you your local Chrome version doesn't match the Chromedriver version from the
 ```
 yarn run webdriver-update --versions.chrome=77.0.3865.120
 ```
+Or if you are using OSX, run:
```

Comment:
Nit:  `OS X` is now `macOS`.  Perhaps we should use the current name here and for the task?

Reply:
I could also:

```json
// package.json
"webdriver-update-macos": "CHROME_VERSION=$(/Applications/Google\\ Chrome.app/Contents/MacOS/Google\\ Chrome --version) && yarn webdriver-update --versions.chrome=\"${CHROME_VERSION}\"",
"webdriver-update-osx": "yarn webdriver-update-osx"
```
too much?  I guess its been `macOS` since 2016.  Should prob get over that change.

Reply:
I'd just use `webdriver-update-mac`. I don't see a reason for an alias

Reply:
yeah didn't bother with the alias.